cogboard-webapp/src/components/BoardCard/index.js

```diff
@@ -24,6 +28,51 @@ const BoardCard = ({ boardData, className }) => {
   const dispatch = useDispatch();
   const isAdmin = useSelector(({app}) => app.isAdmin);
 
+  const ref = useRef(null);
+  const isLoggedIn = useSelector(({ app }) => !!app.jwToken);
+  const [{ isDragging }, drag] = useDrag({
+    item: { type: ItemTypes.BOARD, id, index },
+    canDrag: isLoggedIn,
+    collect: monitor => ({
+      isDragging: monitor.isDragging()
+    })
+  });
+  const [{ isOver }, drop] = useDrop({
```

Comment:
As a backend guy 4 return statements in one method are bit strange in my standards.

Reply:
fixed